dist/svg.connectable.js

```diff
@@ -0,0 +1,509 @@
+        options = {};
+    }
+
+    container = options.container || container;
+    var elmSource = this;
+    markers = options.markers || markers;
+    options.k = options.k || 100;
+
+    var marker = markers.marker(10, 10)
+      , markerId = "triangle-" + Id()
+      , line = container.path().attr("marker-end", "url(#" + markerId + ")").fill("none")
+      ;
+
+    marker.attr({
+        id: markerId,
+        viewBox: "0 0 10 10",
+        refX: "0",
+        refY: "5",
+        markerUnits: "strokeWidth",
+        markerWidth: "4",
+        markerHeight: "5"
+    });
+
+    marker.path().attr({
+        d: "M 0 0 L 10 5 L 0 10 z"
+    });
+
+    // Source and target positions
+    var sPos = {}
+      , tPos = {}
+      ;
+
+    // Append the SVG elements
+    con.source = elmSource;
+    con.target = elmTarget;
+    con.line = line;
+    con.marker = marker;
+
+    SetOrGet(_connections, con.source.id(), []).push(con);
+    SetOrGet(_connections, con.target.id(), []).push(con);
+
+    SetOrGet(_betweenTwoBubbles, {
+        toString: function () {
+            var ids = [con.source.id(), con.target.id()]
+              , id1 = ids.join("->")
+              , id2 = ids.reverse().join("->")
+              ;
+
+            con._ = con.id = id1;
+
+            if (_betweenTwoBubbles[id2]) {
+                con._ = id2;
+                return id2;
+            }
+
+            con.id = id1;
+            return id1;
+        }
+    }, []).push(con);
+
+    /**
+     * computeLineCoordinates
+     * The function that computes the new coordinates.
+     * It can be overriden with a custom function.
+     *
+     * @name computeLineCoordinates
+     * @function
+     * @param {Connectable} con The connectable instance.
+     * @return {Object} An object containing the `x1`, `x2`, `y1` and `y2` coordinates.
+     */
+    con.computeLineCoordinates = function (cons) {
+
+        var output = []
+          , l = cons.length
+          ;
+
+        IterateObject(cons, function (con, i) {
+
+            var sT = con.source.transform()
+              , tT = con.target.transform()
+              , sB = con.source.bbox()
+              , tB = con.target.bbox()
+              , x1 = sT.x + sB.width / 2
+              , y1 = sT.y + sB.height / 2
+              , x2 = tT.x + tB.width / 2
+              , y2 = tT.y + tB.height / 2
+              , cx = (x1 + x2) / 2
+              , cy = (y1 + y2) / 2
+              , dx = (x1 - x2) / 2
+              , dy = (y1 - y2) / 2
+              , dd = null
+              , out = {
+                    x1: x1
+                  , y1: y1
+                  , x2: x2
+                  , y2: y2
+                  , ex: x1
+                  , ey: y1
+                }
+              ;
+
+            if (i !== (l - 1) / 2) {
+                dd = Math.sqrt(dx * dx + dy * dy);
+                out.ex = cx + dy / dd * options.k * (i - (l - 1) / 2);
+                out.ey = cy - dx / dd * options.k * (i - (l - 1) / 2);
```

Comment:
Need some help here. :sos: 

When we connect `A` to `B` and `B` to `A` this will fail: both curved lines will be on the same side. Anybody care to take a look?

/cc @ctzurcanu @jillix/devs-write